Add March 2024 newsletter (#275)
readthedocs/website: content/posts/newsletter-march-2024.md

title: Read the Docs Newsletter - March 2024
date: March 13, 2024
description: Company updates and new features from the last month, current focus, and upcoming features.
category: Newsletter
tags: newsletter, community
authors: Eric Holscher
status: published
image: /images/mar-2024.jpg
image_credit: Photo by <a href="https://unsplash.com/@catrionaobrian?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Ekaterina Novitskaya</a> on <a href="https://unsplash.com/photos/white-cherry-blossom-in-bloom-during-daytime-kcwb2wDxQFM?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>

## News and updates

> - We added [support for "latest"](https://about.readthedocs.com/blog/2024/02/introducing-latest-aliases/) in a number of build tools, allowing you to always be up to date.

Reviewer: ```suggestion
- We added [support for "latest"](https://about.readthedocs.com/blog/2024/02/introducing-latest-aliases/) in all of the build tools, allowing you to always be up to date.
```
